components/loading-spinner/README.md

```diff
@@ -0,0 +1,18 @@
+# Loading Spinner
+
+## d2l-loading-spinner
+
+![Loading Spinner](./screenshots/loading-spinner.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/loading-spinner/loading-spinner.js';
+</script>
+
+<d2l-loading-spinner></d2l-loading-spinner>
+```
+
+**Properties:**
+
+- `color` (String): Color of the animated bar (default is `--d2l-color-celestine`)
+- `size` (Number): Height and width (`px`) of the spinner (default is `50px`)
```

Comment:
Nit: should probably say "default is `50`" since the value doesn't include the "px"

Reply:
Good point... will update.